man/adoc/bpftrace.adoc

```diff
@@ -3340,6 +3352,14 @@ Default: 0
 
 For user space symbols, symbolicate lazily/on-demand (1) or symbolicate everything ahead of time (0).
 
+=== license
+
+Default: <unset>
+
+The license covering the script's source code.
+
+This license string is passed to the kernel where it is used to determine what functionality can be used.
```

Comment:
Should we specify that it can only be in the set of allowed strings?
https://elixir.bootlin.com/linux/latest/source/include/linux/license.h